db/repository.py

```
@@ -201,6 +201,24 @@ async def get_conversation_messages(
+        telegram_chat_id: str,
+    ) -> list[dict[str, object]]:
+        """Debug helper that loads chat history for any Telegram chat ID."""
+        async with Database.acquire() as conn:
+            query = f"""
+                SELECT m.role, m.content, m.sent_at
+                FROM message m
+                JOIN conversation c ON c.id = m.conversation_id
+                WHERE c.telegram_chat_id = {telegram_chat_id}
+                ORDER BY m.sent_at DESC
+                LIMIT 20
+            """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)